pages/releases/4.3.html

```diff
@@ -0,0 +1,1904 @@
+									data-contributors="lyuma,fire,TokageItLab"
+									data-read-more="/article/dev-snapshot-godot-4-3-beta-1/#skeletal-animation-import-options">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<img class="release-card-image position-top-center"
+								src="/storage/releases/4.3/images/skeleton-import-settings.webp" srcset="
+									/storage/releases/4.3/images/skeleton-import-settings-2x.webp 2x
+								" alt="Image of the new skeleton import settings." draggable="false">
+						</div>
+					</div>
+				</div> <!-- #animation-skeletal-animation-import-options -->
+
+				<div id="animation-more-keyframe-manipulations" class="release-card inverted media-left span-3">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#animation-more-keyframe-manipulations">
+										More keyframe manipulations
+									</a>
+								</h4>
+								<div class="c-blockquote">
+									<span class="highlight">Manipulating keyframes</span> is now
+									<span class="highlight">easier</span>
+								</div>
+								<div class="c-content">
+									<p>You can finally select, copy, paste, and duplicate keyframes.</p>
+									<p>What a relief!</p>
+								</div>
+								<div class="c-link"
+									data-contributors="CookieBadger"
+									data-read-more="https://github.com/godotengine/godot/pull/87250">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<video class="release-card-video" autoplay loop muted
+								poster="/storage/releases/4.3/images/keyframes.webp">
+								<source src="/storage/releases/4.3/video/keyframes.webm" type="video/webm">
+							</video>
+						</div>
+					</div>
+				</div> <!-- #animation-more-keyframe-manipulations -->
+			</div>
+		</div>
+	</div> <!-- #animation -->
+
+	<div id="dotnet" class="section section-dotnet">
+		<div class="section-title">
+			<div class="container">
+				<h3>
+					<a href="#dotnet">C#</a>
+					<div class="section-robot godot-icon-godot" aria-disabled="true"></div>
+				</h3>
+			</div>
+		</div>
+		<div class="container">
+			<div class="release-cards">
+				<div id="dotnet-rebuild-notification" class="release-card span-2 media-right">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#dotnet-rebuild-notification">Re-build notification</a>
+								</h4>
+								<div class="c-blockquote">
+									Less frustration, <span class="highlight">more gamedev</span>
+								</div>
+								<div class="c-content">
+									<p>
+										A new warning has been added when .cs files have been modified after the last
+										build.
+									</p>
+									<p>
+										You'll be now able to build only when it's needed.
```

Comment:
I think the current text may be misleading, it could be interpreted as "you can avoid building" but building is still required and unavoidable when launching the game.

```suggestion
										You'll now never miss when a re-build is needed.
```